src/main/java/com/mars/review/ui/ReviewController.java

```diff
@@ -0,0 +1,30 @@
+package com.mars.review.ui;
+
+import com.mars.review.application.ReviewService;
+import com.mars.review.dto.ReqCreateReview;
+import com.mars.review.dto.ResReview;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+public class ReviewController {
+    private final ReviewService reviewService;
+
+    @Autowired
+    public ReviewController(ReviewService reviewService) {
+        this.reviewService = reviewService;
+    }
+
+    @PostMapping(value = "/api/review")
+    public Long save(@RequestBody ReqCreateReview req) {
+        return this.reviewService.save(req);
+    }
+
+    @GetMapping(value = "/api/review/{id}")
+    public List<ResReview> findByMemberId(@PathVariable(name = "id") Integer id){
```

Comment:
pk로 사용하는 id는 Long타입으로 선언해주세요~ 메모리 크기 관련 이슈를 찾아보면 좋을 것 같습니다
```suggestion
    public List<ResReview> findByMemberId(@PathVariable(name = "id") Long id){
```

Reply:
넵